### DISCUSSION TITLE
Error: globalThis.__publicAssetsURL

### DISCUSSION THREAD

**Discussion by ariefsn** [nuxt/nuxt#16295] [Questions]
hi,
I have issue after build and run the output.
![image](https://user-images.githubusercontent.com/33569737/185641511-321904f4-d7fd-419f-97fd-853daeb780d7.png)
do you guys any experience with this?
node v16.17.0
nuxt v3.0.0-rc.6

Thanks

**Comment by ariefsn** [comment]
Up 😢

**Comment by safiullahsarhandi** [comment]
facing same error its working on local environment but not on dedicated server
any solution?

**Reply by safiullahsarhandi** [reply]
i found the solution like the nitro version is different. its working fine on ` 0.5.4 ` for me you can check version related info with 
` npx nuxi info `

**Comment by kissu** [comment]
Same issue here: https://stackoverflow.com/q/74568834/8816585

Could you please provide a reproduction @ariefsn? 🙏🏻
